src/notifier/index.js

```diff
@@ -0,0 +1,173 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import sendInBlue from 'sib-api-v3-sdk';
+const defaultClient = sendInBlue.ApiClient.instance;
+
+const authentication = defaultClient.authentications['api-key'];
+authentication.apiKey = process.env.SENDINBLUE_API_KEY;
+
+const apiInstance = new sendInBlue.SMTPApi();
+const contactsInstance = new sendInBlue.ContactsApi();
+
+const LIST_FOLDER_ID = Number.parseInt(process.env.NODE_ENV === 'test' ? process.env.LIST_TEST_FOLDER_ID : process.env.LIST_FOLDER_ID, 10);
+const UPDATE_TEMPLATE_ID = Number.parseInt(process.env.UPDATE_TEMPLATE_ID, 10);
+const ERROR_TEMPLATE_ID = Number.parseInt(process.env.ERROR_TEMPLATE_ID, 10);
+const BASE_URL = process.env.BASE_URL;
+
+const serviceProvidersMailingLists = {};
+let serviceProviders;
+let documentTypes;
+
+let initialized = false;
+export async function init(passedServiceProviders, passedDocumentTypes) {
+  if (!initialized) {
+    serviceProviders = passedServiceProviders;
+    documentTypes = passedDocumentTypes;
+    return bootstrapMailingLists().then(() => {
+      initialized = true;
+    });
+  }
+
+  return Promise.resolve();
+}
+
+async function bootstrapMailingLists() {
+  const listsPromises = [];
+
+  Object.keys(serviceProviders).forEach((serviceProviderId) => {
+    const { documents, serviceProviderName } = serviceProviders[serviceProviderId];
+
+    serviceProvidersMailingLists[serviceProviderId] = {};
+
+    listsPromises.push(generateServiceProviderList({ serviceProviderName, serviceProviderId }));
+
+    Object.keys(documents).forEach((documentId) => {
+      listsPromises.push(generateDocumentsLists({
+        serviceProviderName,
+        serviceProviderId,
+        documentName: documentTypes[documentId].name,
+        documentId,
+      }));
+    });
+  });
+
+  return Promise.all(listsPromises);
+}
+
+async function generateServiceProviderList({ serviceProviderName, serviceProviderId }) {
```

Comment:
This was really ambiguous for me at first read.

```suggestion
async function generateServiceProviderMailingList({ serviceProviderName, serviceProviderId }) {
```